Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
 Section 5
 1: Each House shall be the Judge of the Elections, Returns and Qualifications of its own Members, and a Majority of each shall constitute a Quorum to do Business; but a smaller Number may adjourn from day to day, and may be authorized to compel the Attendance of absent Members, in such Manner, and under such Penalties as each House may provide.
 
-2: Each House may determine the Rules of its Proceedings, punish its Members for disorderly Behaviour, and, with the Concurrence of two thirds, expel a Member.
+2: Each House may determine the Rules of its Proceedings, punish its Members for disorderly Behavior, and, with the Concurrence of two thirds, expel a Member, who shall when so expelled be ineligible to return until one full term of Representation has elapsed.
 
-3: Each House shall keep a Journal of its Proceedings, and from time to time publish the same, excepting such Parts as may in their Judgment require Secrecy; and the Yeas and Nays of the Members of either House on any question shall, at the Desire of one fifth of those Present, be entered on the Journal.
+3: Each House shall keep a Journal of its Proceedings, and within 3 months publish the same, excepting such Parts as may in their Judgement require Secrecy. All matters which are so excepted shall be published within 15 Years of the Proceedings, though this may be extended a further 6 years in times of declared War.
 
 4: Neither House, during the Session of Congress, shall, without the Consent of the other, adjourn for more than three days, nor to any other Place than that in which the two Houses shall be sitting.
 
 Section 6
-1: The Senators and Representatives shall receive a Compensation for their Services, to be ascertained by Law, and paid out of the Treasury of the United States.6 They shall in all Cases, except Treason, Felony and Breach of the Peace, be privileged from Arrest during their Attendance at the Session of their respective Houses, and in going to and returning from the same; and for any Speech or Debate in either House, they shall not be questioned in any other Place.
+1: The Senators and Representatives shall receive a Compensation for their Services, to be ascertained by Law, and paid out of the Treasury of the State which they represent. They shall in all Cases, except Treason, Felony and Breach of the Peace, be privileged from Arrest during their Attendance at the Session of their respective Houses, and in going to and returning from the same; and for any Speech or Expression in either House, they shall not be questioned by State government or the Government of these United States, but will be answerable only to the House wherein they serve.
```

Comment:
The significant change here is the source of pay for the Representatives and Senators. Let them be reminded that they serve their District and their State, not the Federal Government.